README.md

```diff
@@ -1,184 +1,79 @@
-<img
-    alt="Binder"
-    valign="bottom"
-    height="21px"
-    src="https://mybinder.org/badge_logo.svg"/>
-</a>
-
-# Quick Example
-
-## `example_app/components.py`
-
-This is where you'll define your [IDOM](https://github.com/idom-team/idom) components. Ultimately though, you should
-feel free to organize your component modules as you wish. Any components created will ultimately be referenced
-by Python dotted path in `your-template.html`.
-
-```python
-from idom import component, html
-from django_idom import IdomWebsocket
-
-# Components are CamelCase by ReactJS convention
-@component
-def Hello(websocket: IdomWebsocket, greeting_recipient: str):
-    return html.h1(f"Hello {greeting_recipient}!")
-```
-
-## [`example_app/templates/your-template.html`](https://docs.djangoproject.com/en/dev/topics/templates/)
-
-In your templates, you may add IDOM components into your HTML by using the `idom_component`
-template tag. This tag requires the dotted path to the component function.
-
-Additonally, you can pass in keyworded arguments into your component function.
-
-In context this will look a bit like the following...
-
-```jinja
-{% load idom %}
-
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Example Project</title>
-  </head>
-
-  <body>
-    {% idom_component "my_django_project.example_app.components.Hello" greeting_recipient="World" %}
-  </body>
-</html>
-```
-
-# Installation
-
-Install `django-idom` via pip.
+<!--header-start-->
 
-```bash
-pip install django-idom
-```
-
-You'll also need to modify a few files in your Django project.
-
-## [`settings.py`](https://docs.djangoproject.com/en/dev/topics/settings/)
-
-In your settings you'll need to add `channels` and `django_idom` to [`INSTALLED_APPS`](https://docs.djangoproject.com/en/dev/ref/settings/#std:setting-INSTALLED_APPS).
-
-```python
-INSTALLED_APPS = [
-  ...,
-  "channels",
-  "django_idom",
-]
-
-# Ensure ASGI_APPLICATION is set properly based on your project name!
-ASGI_APPLICATION = "my_django_project.asgi.application"
-```
-
-**Optional:** You can also configure IDOM settings.
+# Django IDOM &middot; [![Tests](https://github.com/idom-team/django-idom/workflows/Test/badge.svg?event=push)](https://github.com/idom-team/django-idom/actions?query=workflow%3ATest) [![PyPI Version](https://img.shields.io/pypi/v/django-idom.svg)](https://pypi.python.org/pypi/django-idom) [![License](https://img.shields.io/badge/License-MIT-purple.svg)](https://github.com/idom-team/django-idom/blob/main/LICENSE)
 
-```python
-# If "idom" cache is not configured, then we'll use "default" instead
-CACHES = {
-  "idom": {"BACKEND": ...},
-}
+<!--header-end-->
+<!--intro-start-->
 
-# Maximum seconds between two reconnection attempts that would cause the client give up.
-# 0 will disable reconnection.
-IDOM_WS_MAX_RECONNECT_TIMEOUT: int = 604800
+IDOM connects your Python web framework of choice to a ReactJS frontend, allowing you to create **interactive websites without needing JavaScript!**
```

Comment:
Do you think we should tailor this intro more to a Django audience? For example, changing things like "your web framework of choice" to just "Django"

Reply:
I made a revision to address this, let me know what you think.